### PR TITLE
Show eduroam/local_alias/memberID in members drop-downs.

### PR DESCRIPTION
**Pull request by mstratford**
Fixes: UniversityRadioYork/Issues#56, #592 

Looks like the issues I was having before was due to caching, the code is working as it should now (after ignoring it for ages). It chooses eduroam first, followed by localAlias, followed by memberID as least preferred.  👍

### CODE REVIEW

**Comment by LloydW93** [comment]
I know this conversation has been going on for basically forever now, so I'll try not to ask for all the new functionality, but how about not showing any of these additional fields if there's no duplicates?

Can't remember if I've asked this before, but is *any* MyRadio user able to see any other user's email today?

**Comment by adfw** [comment]
A 'normal' user can't, committee positions generally can see emails

**Comment by LloydW93** [comment]
Then it's worth being aware that we're making information more "public" than it was before by doing this. Ofc, technically anyone at UoY can sign up whenever they want, but *also* anyone at UoY can also just type the name into gmail, so...

**Comment by mstratford** [comment]
Erm, well it comes directly from a SQL query, to a JSON string to the list you get on screen, without some really clever stuff, I can’t see how I could do that… (Bearing in mind my SQL knowledge isn’t exactly huge.)

**Comment by LloydW93** [comment]
I can't see a way of doing it without a fair bit of work either (or at least, without trying to remember how typeahead works), so meh.

**Comment by adfw** [comment]
I suppose on the whole, these member-select dropdowns are mostly restricted by permissions anyway

File: src/Public/js/myradio.form.js [hunk 1]
```diff
@@ -78,7 +75,15 @@ window.MyRadioForm = {
                                     //Fix typeahead not showing after hiding
                                     //TODO: Report this @ https://github.com/twitter/typeahead.js/
                                     $('input:focus').parent().children('.tt-dropdown-menu').removeClass('hidden');
-                                    return '<p>' + i.fname + ' ' + i.sname + '</p>';
+                                    var identity
```
Comment (LordAro) [inline src/Public/js/myradio.form.js]: don't like the lack of assignment here - `var identity = '';` ?

Comment (mstratford) [inline src/Public/js/myradio.form.js]: Done & retested.

**Comment by mstratford** [comment]
Fixed :) Using the lack of much TypeAhead/Bloodhound documentation, all I know is that it is possible to hide duplicates client side, but that doesn't count if the different details are provided through JSON first.

**Pull request merged**